tools/travis/build.sh

```diff
@@ -11,7 +11,7 @@ tools/build/scanCode.py .
 
 cd $ROOTDIR/ansible
 
-ANSIBLE_CMD="ansible-playbook -i environments/travis"
+ANSIBLE_CMD="ansible-playbook -i environments/local"
```

Comment:
Is this required, or we changed our minds to re-use `local` for travis also?

As a side note, I think using "local" as the name of the environment now becomes even more vague.
I think we should rename it to `environment/ubuntu-single-box`